test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -34,7 +34,7 @@ public abstract partial class BaseTestBaseline : BaseTestClass
 
         protected BaseTestBaseline(ITestOutputHelper output) : base(output)
         {
-#if NET462
+#if NET461
```

Comment:
NOTE: I just learned about `VersionlessImplicitFrameworkDefine` in the .NET Core SDK - https://github.com/dotnet/sdk/pull/2073. It appears this can be changed to `#if NETFRAMEWORK`

Reply:
I will add this change in my next PR